### ISSUE TITLE
Ability to sort paket files on install

### ISSUE THREAD

**Issue by BlythMeister** [fsprojects/Paket#3532]
When running install, sort paket.reference and paket.dependencies files so they are alphabetical.

Taking the sorting logic from https://github.com/BlythMeister/PaketSorter and baking it into paket directly.

**Comment by forki** [comment]
what's the idea?

**Comment by BlythMeister** [comment]
I made this paket sorting tool.
It was suggested that paket could do this automatically to make all paket files alphabetical.  So I raised this.

**Comment by forki** [comment]
yeah I read that part, but I don't understand what it's sorting and why. Can you please elaborate on the idea. Thank you!

**Comment by BlythMeister** [comment]
Ahh right.

It sorts paket reference and dependency files alphabetically...because I'm very particular and like things neat and tidy 🤣

**Comment by forki** [comment]
Sorts the contents? That's not a stable operation

**Comment by BlythMeister** [comment]
It parses the lines in the files and attempts to sort them, so takes into account groups etc.

But if this is not 100% safe, then fair enough, don't bake it in 🙂
Or maybe a new command which is a best effort in the way that simplify is...

**Comment by forki** [comment]
Yes. A cleanup command would be appreciated. Let's make it explicit

**Comment by BlythMeister** [comment]
I wonder if "cleanup" should also run simplify logic...

**Comment by forki** [comment]
I think simplify should sort. But a command that just sorts may be useful as well
